lib/dsprot/include/bss.h

```diff
@@ -0,0 +1,8 @@
+#ifndef BSS_H
+#define BSS_H
+
+#include "nitro_types.h"
+
+extern u8 BSS[4];
+
+#endif
```

Comment:
also please include a comment with every endif saying what is ended
(i.e. // DSPROT_BSS_H)
(obvious here but can be harder in more complex files)